phenotrex/util/external_data.py

```diff
@@ -0,0 +1,59 @@
+import io
+import gzip
+from typing import Dict, Tuple
+import urllib.request
+import ssl
+from urllib.error import HTTPError
+
+from phenotrex.util.logging import get_logger
+
+
+class Eggnog5TextAnnotator:
+
+    BASE_PATH = "http://eggnog5.embl.de/download/eggnog_5.0/per_tax_level"
+
+    def __init__(self):
+        self._known_taxa = {}
+        self.logger = get_logger(self.__class__.__name__)
+
+    @staticmethod
+    def _download_and_zcat(http_path: str) -> str:
+        try:
+            response = urllib.request.urlopen(
+                http_path, timeout=5, context=ssl._create_unverified_context()
+            )
```

Comment:
Can probably revert that, as discussed on slack. The eggnog path is anyway plain http, and apparently does not even support https